### PR TITLE
Upgrade Sudo to 1.9.10

### PR DESCRIPTION
**Pull request by jslobodzian**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**YES/NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

File: SPECS/sudo/sudo.spec [hunk 1]
```diff
@@ -1,18 +1,23 @@
 Summary:        Sudo
 Name:           sudo
-Version:        1.9.5p2
-Release:        3%{?dist}
+Version:        1.9.10
```
Comment (mfrw) [inline SPECS/sudo/sudo.spec:3]: I think we missed updating `cgmanifest.json`

**Review by mfrw — APPROVED** [review]
LGTM.
One nit about modifying `cgmanifest.json`

**Comment by anphel31** [comment]
Can you please also include audit support in sudo with the following changes:
BuildRequires:  audit-devel
Requires:  audit-libs
./configure --with-linux-audit

File: SPECS/sudo/sudo.spec (hunk 1 above)
Comment (jslobodzian) [reply SPECS/sudo/sudo.spec:3]: Yep.  Fixed,

**Review by nicogbg — APPROVED** [review]

**Review by PawelWMS — APPROVED** [review]

**Pull request merged**